transport/src/main/java/io/netty/channel/ChannelConfig.java

```diff
@@ -183,6 +183,17 @@
     ChannelConfig setRecvByteBufAllocator(RecvByteBufAllocator allocator);
 
     /**
+     * Returns {@code true} if {@link ChannelOption#AUTO_FLUSH} option is set to {@code true}.
+     */
+    boolean isAutoFlush();
+
+    /**
+     * Sets if explicit flushes are not required for this channel. If set to {@code true} every eventloop iteration
+     * will flush all pending writes. Explicit flushes will still be honored. The default value is {@code false}.
+     */
+    ChannelConfig setAutoFlush(boolean autoFlush);
```

Comment:
Yeah I wasn't sure about breaking backward compatibility. I will remove the changes that breaks backward compatibility in the way you suggested.